lib/constants.go

```diff
@@ -911,9 +917,24 @@ func (params *DeSoParams) EnableRegtest() {
 	// Set the PoS default jail inactive validator grace period epochs to 3.
 	params.DefaultJailInactiveValidatorGracePeriodEpochs = 3
 
+	// Check for consensus transition every second.
+	params.FastHotStuffConsensusTransitionCheckDuration = 1 * time.Second
+
 	// In regtest, we start all the fork heights at zero. These can be adjusted
 	// for testing purposes to ensure that a transition does not cause issues.
 	params.ForkHeights = RegtestForkHeights
+	if isAcceleratedRegtest {
+		params.ForkHeights.ProofOfStake2ConsensusCutoverBlockHeight = 30
+		params.DefaultEpochDurationNumBlocks = uint64(5)
+		params.DefaultBlockProductionIntervalMillisecondsPoS = 1000 // 1s
+		params.DefaultTimeoutIntervalMillisecondsPoS = 2000         // 2s
+		newSeedBalance := &DeSoOutput{
```

Comment:
This is so smart. My hacked version of core never had this and it was so annoying.